services/venice-router/src/main/java/com/linkedin/venice/router/api/DictionaryRetrievalService.java

```diff
@@ -137,6 +139,27 @@ public void handleStoreChanged(Store store) {
     }
   };
 
+  enum LogLevel {
+    INFO, WARN
+  };
+
+  private final RedundantExceptionFilter redundantExceptionFilter = new RedundantExceptionFilter(
+      RedundantExceptionFilter.DEFAULT_BITSET_SIZE,
```

Comment:
Shall we create a new constructor for `RedundantExceptionFilter`, which will take these default params instead of the users of this class to do that?

Reply:
Cleaner suggestion. Done.